extensions/kubernetes/openshift/deployment/src/main/java/io/quarkus/openshift/deployment/OpenshiftDevConsoleProcessor.java

```diff
@@ -13,4 +27,59 @@ DevConsoleRouteBuildItem builder() {
         return new DevConsoleRouteBuildItem("deploy", "POST",
                 new RebuildHandler(Collections.singletonMap("quarkus.kubernetes.deploy", "true")));
     }
+
+    @BuildStep(onlyIf = IsDevelopment.class)
+    CardPageBuildItem create(List<AvailableContainerImageExtensionBuildItem> extensions) {
+        // Get the list of builders
+        JsonArray array = extensions.stream().map(AvailableContainerImageExtensionBuildItem::getName).sorted()
+                .collect(JsonArray::new, JsonArray::add, JsonArray::addAll);
+
+        CardPageBuildItem card = new CardPageBuildItem();
+        card.addBuildTimeData("builderTypes", array);
+        card.addPage(Page.webComponentPageBuilder()
+                .title("Deploy to OpenShift")
+                .componentLink("qwc-openshift-deployment.js")
+                .icon("font-awesome-solid:box"));
+        return card;
+    }
+
+    @BuildStep(onlyIf = IsDevelopment.class)
+    JsonRPCProvidersBuildItem createJsonRPCServiceForContainerBuild() {
+        DevConsoleManager.register("openshift-deployment-action", build());
+        return new JsonRPCProvidersBuildItem(OpenshiftDeploymentJsonRpcService.class);
+    }
+
+    private Function<Map<String, String>, String> build() {
+        return (map -> {
+
+            QuarkusBootstrap existing = (QuarkusBootstrap) DevConsoleManager.getQuarkusBootstrap();
+            try (TempSystemProperties properties = new TempSystemProperties()) {
+                properties.set("quarkus.kubernetes.deploy", "true");
+                for (Map.Entry<String, String> arg : map.entrySet()) {
+                    properties.set(arg.getKey(), arg.getValue());
+                }
+                System.out.println("Running build..." + properties + " / " + map);
```

Comment:
Are these `System.out` statements intended to stay?

Reply:
Nope, that is a left-over from my long debugging session.